docs/using/tutorial/Vagrantfile.template

```diff
@@ -17,11 +52,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "node1" do |node1|
     node1.vm.network :private_network, :ip => "172.16.255.250"
     node1.vm.hostname = "node1"
+    node1.vm.provision "shell", inline: $bootstrap_common, privileged: true
+    node1.vm.provision "shell", inline: $bootstrap_control, privileged: true
+    node1.vm.provision "shell", inline: $bootstrap_node1_cert, privileged: true
+    node1.vm.provision "shell", inline: $bootstrap_node, privileged: true
   end
 
   config.vm.define "node2" do |node2|
     node2.vm.network :private_network, :ip => "172.16.255.251"
     node2.vm.hostname = "node2"
+    node2.vm.provision "shell", inline: $bootstrap_common, privileged: true
+    node2.vm.provision "shell", inline: $bootstrap_node2_cert, privileged: true
+    node2.vm.provision "shell", inline: $bootstrap_node, privileged: true
   end
 
+  # Don't use a shared folder.
+  config.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
Probably not worth it now but this could be split out into another PR (with the vbguest change).